content/ch-quantum-hardware/randomized-benchmarking.ipynb

```diff
@@ -52,7 +52,7 @@
     "from IPython import display\n",
     "\n",
     "# Import the RB Functions\n",
-    "import qiskit.ignis.verification.randomized_benchmarking as rb\n",
+    "import qiskit_experiments.library.randomized_benchmarking as rb\n",
```

Comment:
Same problem here; when running I get 
```
AttributeError: module 'qiskit_experiments.library.randomized_benchmarking' has no attribute 'randomized_benchmarking_seq'
```

If it only takes a couple of lines to get it working then we can do that, for any larger changes we should leave it using ignis and do a re-write in the new textbook.